pages/index.mdx

```diff
@@ -47,12 +48,13 @@ export const BannerWrapper = styled.div`
 
 export const ContentContainer = styled.div`
   padding: 60px;
+  > h3 {
```

Comment:
This may be a hacky workaround. 

I tried ditching importing Typography.H3 completely, but the margins weren't working with customer styles 🤔 I'll put a pin on this one to revisit.